clients/admin-ui/cypress/e2e/user-management.cy.ts

```diff
@@ -258,14 +258,11 @@ describe("User management", () => {
           cy.getByTestId("row-fidesctl_system").within(() => {
             cy.getByTestId("unassign-btn").click();
           });
-          cy.getByTestId("remove-fidesctl_system-confirmation-modal").within(
-            () => {
-              cy.getByTestId("continue-btn").click({ force: true });
-            }
-          );
-          cy.wait("@removeUserManagedSystem").then((interception) => {
-            const { url } = interception.request;
-            expect(url).contains("fidesctl_system");
+          cy.getByTestId("save-btn").click();
+
+          cy.wait("@updateUserManagedSystems").then((interception) => {
+            const { body } = interception.request;
+            expect(body).to.eql([]);
```

Comment:
nice, I think we should take this test suite a little farther too
1. Let's remove the intercept of `removeUserManagedSystem` since we're no longer calling that endpoint and don't need it
2. for the tests under `in modal`, when we do a `cy.wait("@updateUserManagedSystems")`, let's also, right before it, do a `cy.wait("@updatePermission")` so that we can verify both calls are firing. alternatively, can wait for `@updateUserManagedSystems` in the test "can assign a role to a user"—just want a test that shows both calls firing when the save button is clicked.

Reply:
ah okay, so I think this test starts off with 
<img width="258" alt="image" src="https://user-images.githubusercontent.com/24641006/227623365-74e4b3ca-d52f-4e9d-af61-c7d91a891805.png">

and we remove `fidesctl_system`. the update request should still have the two systems, so let's change the expected payload from `[]` to probably `["demo_analytics_system", "demo_marketing_system"]`